.github/workflows/nexus-feature-branch.yml

```diff
@@ -131,10 +137,10 @@ jobs:
           } | Set-Content -Path 'apps/mudita-center/package.json'
       - name: Setup depedencies
         run: npm run setup
-#      - name: Copy electron-builder environment file for Windows
-#        if: matrix.os == 'Windows'
-#        run: |
-#          copy C:\actions-runner\envs\electron-builder.env apps/mudita-center/electron-builder.env
+      #      - name: Copy electron-builder environment file for Windows
+      #        if: matrix.os == 'Windows'
+      #        run: |
+      #          copy C:\actions-runner\envs\electron-builder.env apps/mudita-center/electron-builder.env
```

Comment:
It might break the yml structure and cause the builds failure. It's better not to format the `yml` files

Reply:
Let's remove this part; it is not needed here.